### DISCUSSION TITLE
Transcription of mp3 repeats something about subtitles

### DISCUSSION THREAD

**Discussion by matthiasroder** [openai/whisper#1251] [Q&A] [answered]
I have downloaded a video from youtube with youtube-dl

`youtube-dl --extract-audio --audio-format mp3 -o "%(id)s.%(ext)s" https://www.youtube.com/watch\?v\=sAzxw4XVLUM`

I then checked the resulting mp3 and it sounded fine.

After that I use whisper to transcribe, using this command:

`whisper sAzxw4XVLUM.mp3 --language German`

which gives me the following output:

`/Users/matthias/opt/anaconda3/envs/ai/lib/python3.9/site-packages/whisper/transcribe.py:114: UserWarning: FP16 is not supported on CPU; using FP32 instead
  warnings.warn("FP16 is not supported on CPU; using FP32 instead")
[00:00.000 --> 00:02.000]  Untertitel im Auftrag des ZDF, 2017
[00:30.000 --> 00:40.000]  Untertitel im Auftrag des ZDF
[01:00.000 --> 01:20.000]  Untertitel im Auftrag des ZDF
[01:30.000 --> 01:40.000]  Untertitel im Auftrag des ZDF
[02:00.000 --> 02:28.000]  Untertitel im Auftrag des ZDF
[02:30.000 --> 02:50.000]  Untertitel im Auftrag des ZDF
...`

The m3 begins with silence, then music and then talking.

Any ideas what could cause the problem?

Many thanks!
Matthias

**Comment by mayeaux** [comment, marked as answer]
Check this thread out: https://github.com/openai/whisper/discussions/928

**Reply by matthiasroder** [reply]
Thank you, this was a very helpful hint.
